documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,147 @@
+
+.Migration of virtual machines with encrypted partitions fails during conversion
+
+Occurs for vSphere: Migrations from RHV and OpenStack don't fail, but the encryption key may be missing on the target OCP cluster.
+
+.Snapshots that are created during the migration in OpenStack are not deleted
+
+The Migration Controller service does not delete snapshots that are created during the migration for source virtual machines in OpenStack automatically. Workaround: the snapshots can be removed manually on OpenStack. 
+
+.RHV snapshots are not deleted after a successful migration
+
+The Migration Controller service does not delete snapshots automatically after a successful warm migration of a {rhv-short} VM. Workaround: Snapshots can be removed from RHV instead.
+
+.Migration fails during precopy/cutover while a snapshot operation is executed on the source VM
+
+Some warm migrations from RHV might fail. When running a migration plan for warm migration of multiple VMs from RHV, the migrations of some VMs might fail during the cutover stage. In that case, restart the migration plan and set the cutover time for the VM migrations that failed in the first run. link:https://bugzilla.redhat.com/show_bug.cgi?id=2063531[(BZ#2063531)] link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+
+Warm migration from RHV fails if a snapshot operation is performed on the source VM. If the user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://bugzilla.redhat.com/show_bug.cgi?id=2057459[(BZ#2057459)]
+
+.Cannot schedule migrated VM with multiple disks to more than one storage classes of type hostpath
+
+When migrating a VM with multiple disks to more than one storage classes of type hostpath, it may result in a VM that cannot be scheduled. Workaround: It is recommended to use shared storage on the target OCP cluster.
+
+.Deleting migrated VM does not remove PVC and PV
+
+When removing a VM that was migrated, its PVCs and PVs are not deleted. +
+Workaround: remove the CDI importer pods and then remove the remaining PVCs and PVs. link:https://issues.redhat.com/browse/MTV-492[(MTV-492)]
+
+.PVC deletion hangs after archiving and deleting migration plan
+
+When archiving and deleting a migration plan that failed, its PVCs and PVs are not deleted. Workaround: remove the CDI importer pods and then remove the remaining PVCs and PVs. link:https://issues.redhat.com/browse/MTV-493[(MTV-493)]
+
+.VM with multiple disks may boot from non-bootable disk after migration
+
+VM with multiple disks that was migrated may not be able to boot on the target OCP cluster. +
+Workaround: set the boot order appropriately to boot from the bootable disk. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
```

Comment:
The IBM style guide uses "might" instead of "may" to remove possible ambiguity.  Check the + sign.
Consider:  An uppercase letter following a colon, especially when starting a complete sentence.